Makefile

```diff
@@ -28,42 +33,39 @@ else
-	@oc replace --force -f deploy/cluster_role.yaml -n $(NAMESPACE)
-	@oc replace --force -f deploy/service_account.yaml -n $(NAMESPACE)
-	@oc replace --force -f deploy/role_binding.yaml -n $(NAMESPACE)
-	@cat deploy/role_binding.yaml | sed "s/namespace: cloud-resource/namespace: $(NAMESPACE)/g" | oc replace --force -f -
+setup/service_account: kustomize
+	@-oc new-project $(NAMESPACE)
+	@oc project $(NAMESPACE)
+	@-oc create -f config/rbac/service_account.yaml -n $(NAMESPACE)
+	@$(KUSTOMIZE) build config/rbac | oc replace --force -f -	
 
 .PHONY: code/run/service_account
 code/run/service_account: setup/service_account
-	@oc login --token=$(shell oc serviceaccounts get-token cloud-resource-operator -n ${NAMESPACE})
-	@$(OPERATOR_SDK) up local --namespace=$(NAMESPACE)
-
-operator-sdk-v0.10:
-	@echo $(OPERATOR_SDK_OS)
-	@curl -Lo operator-sdk-v0.10 https://github.com/operator-framework/operator-sdk/releases/download/v0.10.1/operator-sdk-v0.10.1-x86_64-$(OPERATOR_SDK_OS) && chmod +x operator-sdk-v0.10
+	@oc login --token=$(shell oc serviceaccounts get-token cloud-resource-operator -n ${NAMESPACE}) --server=$(shell sh -c "oc cluster-info | grep -Eo 'https?://[-a-zA-Z0-9\.:]*'") --kubeconfig=TMP_SA_KUBECONFIG --insecure-skip-tls-verify=true
+	WATCH_NAMESPACE=$(NAMESPACE) go run ./main.go
 
 .PHONY: code/gen
-code/gen: operator-sdk-v0.10
-	GOROOT=$(shell go env GOROOT) GO111MODULE="on" ./operator-sdk-v0.10 generate k8s
-	GOROOT=$(shell go env GOROOT) GO111MODULE="on" ./operator-sdk-v0.10 generate openapi
-	@go generate ./...
+code/gen: manifests kustomize generate
 
+# Make sure that the previous version and version values are set to correct values.
 .PHONY: gen/csv
 gen/csv:
-	sed -i.bak 's/image:.*/image: quay\.io\/integreatly\/cloud-resource-operator:v$(VERSION)/g' deploy/operator.yaml && rm deploy/operator.yaml.bak
-	@$(OPERATOR_SDK) generate csv --operator-name=cloud-resources --csv-version $(VERSION) --from-version $(PREV_VERSION) --make-manifests=false --update-crds --csv-channel=integreatly --default-channel --verbose
-	@sed -i.bak 's/$(PREV_VERSION)/$(VERSION)/g' deploy/olm-catalog/cloud-resources/cloud-resources.package.yaml && rm deploy/olm-catalog/cloud-resources/cloud-resources.package.yaml.bak
-	@sed -i.bak s/cloud-resource-operator:v$(PREV_VERSION)/cloud-resource-operator:v$(VERSION)/g deploy/olm-catalog/cloud-resources/$(VERSION)/cloud-resources.v$(VERSION).clusterserviceversion.yaml && rm deploy/olm-catalog/cloud-resources/$(VERSION)/cloud-resources.v$(VERSION).clusterserviceversion.yaml.bak
+	@$(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests --kustomize-dir=config/manifests --output-dir packagemanifests/ --version ${VERSION} --default-channel --channel integreatly
+	@sed -i "s/Version = \"${PREV_VERSION}\"/Version = \"${VERSION}\"/g" version/version.go
+	@yq w -i "packagemanifests/${VERSION}/cloud-resource-operator.clusterserviceversion.yaml" metadata.annotations.containerImage ${OPERATOR_IMG}
```

Comment:
@MStokluska can you add yq and version as a dependency in the readme

Reply:
I've added the `yq` to pre-requisites. 
Thanks Austin!